### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by davider80** [kedder/ofxstatement-iso20022#1]
I tried to import my bank statement and I got the following error:

C:\>ofxstatement convert -t iso20022 camt053_20170412091216.xml ynab.ofx
Traceback (most recent call last):
  File "c:\users\xxx\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\xxx\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\xxx\Anaconda3\Scripts\ofxstatement.exe\__main__.py", line 9, in <module>
  File "c:\users\xxx\anaconda3\lib\site-packages\ofxstatement\tool.py", line 150, in run
    return args.func(args)
  File "c:\users\xxx\anaconda3\lib\site-packages\ofxstatement\tool.py", line 128, in convert
    statement = parser.parse()
  File "c:\users\xxx\anaconda3\lib\site-packages\ofxstatement\plugins\iso20022.py", line 38, in parse
    self._parse_statement_properties(tree)
  File "c:\users\xxx\anaconda3\lib\site-packages\ofxstatement\plugins\iso20022.py", line 46, in _parse_statement_properties
    bnk = stmt.find('./s:Acct/s:Svcr/s:FinInstnId/s:BIC', XMLNS)
AttributeError: 'NoneType' object has no attribute 'find'

Bank statement in attachment. ISO-20022 camt.053
[camt053_20170412091216.zip](https://github.com/kedder/ofxstatement-iso20022/files/915697/camt053_20170412091216.zip)

**Comment by kedder** [comment]
@davider80, thanks for sample file. The issue is fixed, ofxstatement-iso20022-0.4.0 is released. Let me know if you have any issues with it.

**Comment by henk84** [comment]
Could we have a test with this file as a sample?
I tried adding it (https://github.com/henk84/ofxstatement-iso20022/tree/feat/more_camt053_test_samples) but the parsing fails:
```
FAILED tests/test_iso20022.py::test_parse_camt053 - ofxstatement.exceptions.ParseError: (0, 'No account currency provided in statement. Please specify one in configuration...
```
I tried adding the currency to the `config` dict but that didn’t work. Any advice?

PS: that branch is based on other changes I’m doing but I don’t think they are relevant to this problem.
